package.json

```diff
@@ -3,9 +3,11 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "spellcheck": "mdspell --en-gb -ranx 'content/**/*.md'",
-    "broken-link": "blcl -ro ./public/",
-    "test": "npm run spellcheck && npm run broken-link"
+    "spellcheck": "mdspell --en-gb --report --ignore-acronyms --ignore-numbers --no-suggestions 'content/**/*.md'",
+    "broken-link-local": "blcl --recursive --ordered --exclude-external ./public/",
+    "broken-link-all": "blcl --recursive --ordered ./public/",
+    "test": "npm run spellcheck && npm run broken-link-local",
+    "ci-test": "npm run spellcheck && npm run broken-link-all"
```

Comment:
To match what you say in the description, you want to be running `broken-link-all` locally rather than on the CI?

Reply:
No; my intent here is to remove the need to be online to run `npm test`, thus that needs to be running the `-local` variant.

> introduces a (default) version of the `test` command

"version" here refers to what I'm changing the "test" command to be -- the previous version being "`npm run spellcheck && npm broken-link`". While I am also introducing a _variant_ of the command for use on the CI, that wasn't what I was intending to refer to, apologies for the confusion.

Reply:
Ok, you are still however running the online tests on the CI, isn't that what youre trying to avoid?

Reply:
There seemed to be a strong opinion on #12 that running the full set of links on CI was preferred, so I left that.
You're right that I would rather not do that.

Reply:
Ah, I assumed this PR was _removing_ the external checking from the CI, that's fine them